.gitignore

```diff
@@ -64,3 +64,4 @@ typings/
 
 karma.conf.js
 protractor.conf.js
+.DS_Store
```

Comment:
Hey, @LotusElize it is a nice move but in your task, you SHOULD work only in YOUR folder.
So, please remove the changes to the files `.gitignore` and `eslintrc.js` from your pull request. That is mean, you have to revert the files to their state in the main branch: 
- https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/.gitignore
- https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/.gitignore